### ISSUE TITLE
Arrow Position in route mode

### ISSUE THREAD

**Issue by toygarlk** [organicmaps/organicmaps#4456]
The position of the arrow in the old versions was both better and easier for the drivers. Now it's at the bottom and a bit useless. It would be very nice if the arrow returns to its original state.

**Comment by vng** [comment]
Do you mean navigating mode? The position arrow always was at the bottom. If not, could you please clarify your issue (screenshots?)

**Comment by toygarlk** [comment]
Yes; in the car, walking, public transport and bike mode. In older versions, the arrow was not at the very bottom border. It was just above the border.

**Comment by vng** [comment]
For the reference https://github.com/organicmaps/organicmaps/pull/3462

@toygarlk Could you please attach some screenshots from your device to verify the problem?

**Comment by toygarlk** [comment]
![Screenshot_20230214_185726_Organic Maps](https://user-images.githubusercontent.com/95170414/218863781-049c2fbf-9abf-433e-9417-d86cb1792a0f.jpg)

Samsung Galaxy S20 FE (SM-G780G) [Android 13]

**Comment by d4f5409d** [comment]
Same here, it's too close

![Screenshot_2023-03-14-13-42-24-04_10a7d9276315e62e56e7300dfec6be06.jpg](https://user-images.githubusercontent.com/76624594/225288390-7d25bbb4-0fc6-4c9f-af00-dc2251f60b94.jpg)

**Comment by pastk** [comment]
The arrow is too low still.

**Comment by pastk** [comment]
There is `mMap.updateMyPositionRoutingOffset(0))` but it doesn't seem to change anything (probably it resets the position for some specific cases).

Looks like it just overrides a default offset set in `drape_frontend/my_position_controller.cpp::kPositionRoutingOffsetY = 104`, but in my quick test adjusting this value didn't have any effect.
